examples/basic/package.json

```diff
@@ -13,12 +13,15 @@
     "pod-install:newarch": "cd ios && RCT_NEW_ARCH_ENABLED=1 bundle exec pod install && cd .."
   },
   "dependencies": {
+    "@expo/metro-runtime": "~3.2.1",
     "@react-native-picker/picker": "2.7.5",
     "expo": "^51.0.14",
     "expo-asset": "^10.0.9",
     "expo-image": "^1.12.12",
     "react": "18.2.0",
+    "react-dom": "18.2.0",
```

Comment:
I think you should add these react-dom and react-native-web in peerDependancies or optionanal dependancy in the main pakage.json

Reply:
can you also add an option to build / launch web version in the scripts object ?

Reply:
Are you sure? The example does need rnw to run in web mode. It won't be pulled by another package.

+ react-native-windows is declared as a dependency, not a peer/optional dependency.

Reply:
sorry, it is exemple package.json ... OK so